### PR TITLE
2699 cars

### PR DESCRIPTION
**Pull request by Holidayy95**
**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

**Review by tom-osborne — CHANGES_REQUESTED** [review]
@Holidayy95 Thanks for this. 

Please can you change the hashes to use backticks

Eg.
```lua
['hash'] = `nimbus`,
```

**Review by tom-osborne — CHANGES_REQUESTED** [review]
Sorry, one more thing... 
I think the following are named incorrectly
`brisos3` should be `brioso3`
`corista` should be `corsita`

File: shared/vehicles.lua [hunk 1]
```diff
@@ -4419,6 +4419,168 @@ QBShared.Vehicles = {
 		['hash'] = `nimbus`,
 		['shop'] = 'air',
 	},
+	['brisos3'] = {          --DLC +set sv_enforceGameBuild 2699 (and below)
```
Comment (BerkieBb) [inline shared/vehicles.lua:4422]: Isn't it `brioso3`?

**Review by BerkieBb — CHANGES_REQUESTED** [review]

**Comment by BerkieBb** [comment]
Ah tom already spotted them

File: shared/vehicles.lua (hunk 1 above)
Comment (tom-osborne) [reply shared/vehicles.lua:4422]: Yep, and `corsita` not `corista`

**Comment by Holidayy95** [comment]
yeah I noticed when checking them out just now, should all be good now

**Review by tom-osborne — APPROVED** [review]

**Review by BerkieBb — APPROVED** [review]

**Pull request merged**